plugins/git/internal-functions

```diff
@@ -215,6 +229,26 @@ cmd-git-sync() {
   fi
 }
 
+cmd-git-unlock() {
+  declare desc="removes the clone folder for an app"
+  local cmd="git:unlock"
+  [[ "$1" == "$cmd" ]] && shift 1
+  declare APP="$1"
+
+  local APP_CLONE_ROOT="$DOKKU_LIB_ROOT/data/git/$APP"
+
+  if [[ -d "$APP_CLONE_ROOT" ]]; then
+    dokku_log_info1 "About to delete $APP_CLONE_ROOT"
+    chown dokku:dokku -R "$APP_CLONE_ROOT"
+    chmod 755 "$APP_CLONE_ROOT"
```

Comment:
Do we need the chown/chmod?

Reply:
Double checking this now. I added commands because at some point was facing `permission-denied` errors.

Reply:
We do not, removed